scripts/update_css.py

```diff
@@ -0,0 +1,94 @@
+"""
+Currently, Prerender injects 1000s of lines of CSS into each file. Shrink utility replaces this with a CSS
+link to a file of all the stuff that was once there.
+
+This script renews the consolidated CSS file with the stuff the OSF pages of tomorrow have injected into them.
+"""
+from bs4 import BeautifulSoup
+import requests
+
+# Reading from HTTP request vs. from a file
+REMOTE = True
+
+# URLs to use if REMOTE. Any example of each type of page will do, really
+page_urls = {
```

Comment:
I guess we do not have to update the CSS through scraping the OSF. If we want to shrink, we can get the CSS from the pages we want to shrink. This seems unnecessary.